src/cleanlab_tlm/utils/config.py

```diff
@@ -0,0 +1,43 @@
+from cleanlab_tlm.internal.constants import (
+    _TLM_DEFAULT_MODEL,
+    _DEFAULT_TLM_QUALITY_PRESET,
+    _TLM_DEFAULT_CONTEXT_LIMIT,
+    _TLM_MAX_TOKEN_RANGE
+)
+
```

Comment:
every function in here needs to appear in a meaningful unit test. That unit test should rely on hardcoded values as little as possible.

E.g. testing get_default_model() can be achieved by initializing TLM w default settings, calling tlm.get_model_name() and asserting that it matches get_default_model() output

Reply:
To test: get_default_context_limit()

You will probably need to hardcode this in the unit test:

CURRENT_DEFAULT_LIMIT = 70000

and then:
1. make a string of tokens whose length is just around CURRENT_DEFAULT_LIMIT - 500 say, and verify tlm.prompt() and tlm.score() successfully run when this string is passed as `prompt`. 

2. make a string of tokens whose length is just around CURRENT_DEFAULT_LIMIT + 500 say, and verify tlm.prompt() and tlm.score()  provide an expected error message when this string is passed as `prompt`.